src/libs/antares/study/parameters.cpp

```diff
@@ -1020,10 +962,33 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     reset();
     // A temporary buffer, used for the values in lowercase
     String value;
-
+    String sectionName;
+    typedef bool (*Callback)(Parameters&, const String&, const String&, const String&, const String&, uint);
+
+    static std::map<String,Callback> sectionAssociatedToKeysProcess = {
+            {"general",&SGDIntLoadFamily_General},
+            {"input",&SGDIntLoadFamily_Input},
+            {"output",&SGDIntLoadFamily_Output},
+            {"optimization",&SGDIntLoadFamily_Optimization},
+            {"other preferences",&SGDIntLoadFamily_OtherPreferences},
+            {"advanced parameters",&SGDIntLoadFamily_AdvancedParameters},
+            {"playlist",&SGDIntLoadFamily_Playlist},
+            {"variables selection",&SGDIntLoadFamily_VariablesSelection},
+            {"seeds",&SGDIntLoadFamily_Seeds}};
```

Comment:
The complete name is **seeds - mersenne twister**

Reply:
OK. Commits b03bbed and b5def8c